lib/v2/components/profile_avatar.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/design/app_theme.dart';
+
+/// PROFILE AVATAR
+///
+/// This class works with http, svg images or just returns a short name
+/// with a colored background
+class ProfileAvatar extends StatelessWidget {
+  final double size;
+  final String image;
+  final String nickname;
+  final String account;
+
+  const ProfileAvatar({
+    this.size = 100,
```

Comment:
dont harcode 100. Make it required to pass the height and width

Reply:
It is a default value that can be overwritten.
I did it this way since it is a global component I assumed that in many other screens it would have this size so instead of writing `size: 100` in each of these screens the value is by default,  if it is you want another value just assigned to it (i.e. `size: 50`)

Reply:
only in the profile they are 100, in other places they are different sizes.

I am asking to not set default value, and force the developer to chose the size. 

Think of it as a library, when you create a library you dont make any assumptions of the size the user will use, you let the user choose the size.

Reply:
Updated 👍